desktop/cmp/appbar/AppBar.ts

```diff
@@ -102,7 +103,8 @@ export const [AppBar, appBar] = hoistCmp.withFactory<AppBarProps>({
                         })
                     ]
                 })
-            ]
+            ],
+            'data-testid': testId
```

Comment:
And in that same discussion with @lbwexler would like to review again if we could avoid having to flip back to `'data-testid'` here when passing to non-Hoist library wrappers.  IIRC we looked at having elem do the conversion, but then the issue is that you have to deal with `data-testid` instead of `testId` within your render methods (I think)

Reply:
Was wondering along the same line if we wanted special handling for say `div` and `span` to take `testId`.  On the fence about this